auth/build.gradle.kts

```diff
@@ -72,8 +76,12 @@ dependencies {
     implementation(Config.Libs.Androidx.fragment)
     implementation(Config.Libs.Androidx.customTabs)
     implementation(Config.Libs.Androidx.constraint)
+    implementation("androidx.credentials:credentials:1.3.0")
+    implementation("androidx.core:core-ktx:1.10.1")
 
     implementation(Config.Libs.Androidx.lifecycleExtensions)
+    implementation("androidx.core:core-ktx:1.13.1")
```

Comment:
We seem to have the same `core-ktx` dependency listed twice with different versions?
Can we remove the older version?